app/controllers/wikies_controller.rb

```diff
@@ -52,6 +52,6 @@ def set_wiky
     end
 
     def wiky_params
-      params.require(:wiky).permit(:title, :body, :user_id)
+      params.require(:wiky).permit(:title, :body, :user_id, :public)
```

Comment:
`:public` should only be a permitted parameter for users that are entitled to make a wiki private. Otherwise there is a flaw that allows standard users to make wikis private. 

We can use the `privatize?` method from `WikiPolicy` to help with this, e.g.:

```
def wiky_params
  permitted_attributes = [:title, :body, :user_id]
  permitted_attributes << :public if policy(@wiky).privatize?
  params.require(:wiky).permit(permitted_attributes)
end
```

Once that's working, consider refactoring it to match the recommended approach here: https://github.com/elabs/pundit#strong-parameters . This is slightly preferable as the `permitted_attributes` decision is more of the responsibility of the WikiPolicy than the WikisController.